ui/react-components/components/ListView/ListView.spec.jsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react'
+import ListView from './ListView'
+import { render, fireEvent } from '@testing-library/react'
+import renderer from 'react-test-renderer'
```

Comment:
We need not introduce ''react-test-renderer" for snapshot testing. react testing library also supports it. Check this -> https://www.leighhalliday.com/introduction-react-testing-library